include/controller.h

```diff
@@ -8,15 +8,39 @@
 
 #include <stdint.h>
 
+#include "joybus.h"
+#include "joybus_n64_accessory.h"
+#include "joypad.h"
+
 /**
  * @addtogroup controller
  * @{
  */
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void execute_raw_command( int controller, int command, int bytesout, int bytesin, unsigned char *out, unsigned char *in );
```

Comment:
This was on my list of outstanding TODOs -- this function is only used by the vrutest example ROM, which is unfinished and undocumented.

The right path forward here would be building a proper subsystem for the VRU/VRS, but that's a research project which is out-of-scope for this PR.

I hope there aren't any projects depending on this function: it's very low-level and basically just a slight convenience wrapper around `joybus_exec`.

I think we should deprecate this function and the associated vrutest example. It's not fully-baked.

Reply:
Even if we eventually decide to keep a "helper wrapper" around for people tinkering with joybus, surely it can't be called `execute_raw_command`. That's really a bad, unprefixed name. It will probably be called `joybus_exec_command` or something right that.

So we should probably deprecate this function ASAP anyway. Can you please add a deprecation warning to it? Probably the message should say to just use `joybus_exec` instead, and then we can revisit it in the future if we decide to introduce a helper.

Reply:
We use it for pixelfx hdmi mod... https://github.com/Polprzewodnikowy/N64FlashcartMenu/blob/5ef09def6902dd9c49a636c77bfe3ac7d89e56f7/src/hdmi/hdmi.c#L53

Reply:
Deprecated `execute_raw_command` and introduced `joybus_send_game_id` + `joybus_clear_game_id` functions in dc65d58a3330cd2e13184442277b88bca5091ce5